test/parser/PythonMetrics.test.ts

```diff
@@ -5,19 +5,117 @@ describe("Python metrics test", () => {
     const pythonTestResourcesPath = "./resources/python/";
 
     describe("parses Python Complexity metric", () => {
+        it("should count loops properly", async () => {
+            await testFileMetrics(pythonTestResourcesPath + "loops.py", FileMetric.complexity, 4);
+        });
+
         it("should count if statements correctly", async () => {
-            await testFileMetrics(pythonTestResourcesPath + "if.py", FileMetric.complexity, 4);
+            await testFileMetrics(
+                pythonTestResourcesPath + "if-statements.py",
+                FileMetric.complexity,
+                6
+            );
+        });
+
+        it("should not count any class declaration", async () => {
+            await testFileMetrics(pythonTestResourcesPath + "classes.py", FileMetric.complexity, 1);
+        });
+
+        it.skip("should count case but no default statements correctly", async () => {
+            await testFileMetrics(
+                pythonTestResourcesPath + "case-statements.py",
+                FileMetric.complexity,
+                3
+            );
+        });
+
+        it("should count functions and methods correctly", async () => {
+            await testFileMetrics(
+                pythonTestResourcesPath + "functions-and-methods.py",
+                FileMetric.complexity,
+                6
+            );
+        });
+
+        it("should not count multiple return statements within functions and methods correctly", async () => {
+            await testFileMetrics(
+                pythonTestResourcesPath + "multiple-return-statements.py",
+                FileMetric.complexity,
+                3
+            );
+        });
+
+        it("should count try-catch-finally properly by only counting the catch-block", async () => {
```

Comment:
can Python have more than 1 catch-block per try-block ? If yes, adding one more catch-block makes this test more precise I think.